### PR TITLE
customer profile printing

### PR DESCRIPTION
**Pull request by dmv-dave86**
This is a working draft for the customer profile printing. Still have some work to do and will move more logic into middleware later, but wanted something working for tomorrow morning's meeting with Amy. Tested in electron and web. This formats the printed page as well.

### CODE REVIEW

File: apps/tech/src/containers/PrintedCustomerProfile/customer_profile.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  .CustomerProfile h1 {
+    font-size: 1.25em;
+    padding: 0;
+    margin: 0;
+    margin-bottom: 2px;
+    font-weight: bold;
+    border-bottom: 2px solid #000000;
+  }
+
+  .CustomerProfile h2{
```
Comment (typesend) [inline apps/tech/src/containers/PrintedCustomerProfile/customer_profile.css]: Space needed

File: apps/tech/src/containers/PrintedCustomerProfile/index.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+const mainStyle = {
+  "fontFamily": "Arial",
+  "marginTop": ".5in",
+  "marginLeft": ".5in",
+  "marginRight": ".5in",
+  "fontSize": "9pt"
+};
+
+const legalTerms = 'I certify that I have read, understand and agree with the contents of this form. I acknowledge that I have received a copy of the declarations and certification statements pertaining to the issuance of a driver license or identification card. I certify (or declare) under penalty of perjury under the laws of the State of California that the information I provided is true and correct.';
```
Comment (typesend) [inline apps/tech/src/containers/PrintedCustomerProfile/index.js]: Please break up this string to multiline concatenated strings.

File: apps/tech/src/helpers/Labels/ch.labels.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  "dlClass": "駕駛執照類別",
+  "dlEndoresement": "消防員認可 ",
+  "cdlClass": "商用駕駛執照類 ",
+  "cdlEndorsements": "商業駕駛執照認可",
+  "cdlCertificates": "商業駕駛執照證書",
+  "cdlSelfCert": "商業駕駛類型",
+  "cdlM": "摩托車除了商業駕駛執照",
+}
+
+export default chineseLabels;
```
Comment (typesend) [inline apps/tech/src/helpers/Labels/ch.labels.js]: Your editor should always add a newline to the last line.

File: apps/tech/src/containers/PrintedCustomerProfile/customer_profile.css (hunk 1 above)
Comment (dmv-dave86) [reply apps/tech/src/containers/PrintedCustomerProfile/customer_profile.css]: fixed

File: apps/tech/src/containers/PrintedCustomerProfile/index.js (hunk 2 above)
Comment (dmv-dave86) [reply apps/tech/src/containers/PrintedCustomerProfile/index.js]: fixed

File: apps/tech/src/helpers/Labels/ch.labels.js (hunk 3 above)
Comment (dmv-dave86) [reply apps/tech/src/helpers/Labels/ch.labels.js]: fixed

**Pull request merged**